charmcraft.yaml

```diff
@@ -27,3 +27,12 @@ parts:
     override-pull: |
       curl -L -O https://github.com/canonical/cos-tool/releases/latest/download/cos-tool-${CRAFT_TARGET_ARCH}
       chmod +x cos-tool-*
+  mimirtool:
+    plugin: dump
+    source: .
+    build-packages:
+      - curl
+    override-pull: |
+      curl -fLo mimirtool https://github.com/grafana/mimir/releases/latest/download/mimirtool-linux-${CRAFT_TARGET_ARCH}
```

Comment:
Is mimir tool released together with mimir? Should we be taking the mimirtool that matches the oci image? Is there an easy way to make sure we match? This can be a separate issue.

Reply:
They are released together; I'm not sure how we would do that, because the Mimir image version is only known at runtime.We can manually pick a version on the same major as the image we